tests/tensorflow/r2.11/tf-resnetrs-imagenet.libsonnet

```diff
@@ -83,18 +71,12 @@ local utils = import 'templates/utils.libsonnet';
   local tpuVm = common.tpuVm,
 
   local functionalTests = [
-    benchmark + accelerator + functional
-    for benchmark in [resnet, resnet_rs]
-    for accelerator in [v2_8, v3_8]
+    resnet_rs + v2_8 + functional,
+    resnet_rs + v3_8 + functional,
   ],
   local convergenceTests = [
-    resnet + v2_32 + convergence + tpuVm,
-    resnet + v3_32 + convergence + tpuVm,
     resnet_rs + v2_32 + convergence + tpuVm + timeouts.Hours(15),
     resnet_rs + v3_32 + convergence + tpuVm + timeouts.Hours(15),
```

Comment:
For tests where you added `timeouts.Hours(15)`, are there also changes to the number of epochs so that tests should train within the 15 hour limit? Otherwise, won't the tests always fail? I highlight this example in particular because from tf 2.12.0 release testing, this test ran for 14+ hours before disconnecting.

Reply:
15 hours are not added from my side (see original [code](https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/tests/tensorflow/nightly/vision-imagenet.libsonnet#L93-L94)). We could have a another bug to track the failure of those tests.

Reply:
Good to know, I was wondering where 15 hours originated. We can submit this PR and find tests that are failing due to this timeout limit and adjust those tests afterwards. This sounds fine to me, but maybe check with another reviewer if its okay to take a reactive stance (fixing tests after merging this PR) rather than a proactive stance (fixing number of epochs for longer duration tests before merging PR).

Reply:
@RissyRan - Are you going to send another PR for fixing number of epochs for conv tests?

Reply:
Sure, I will create a bug and another PR.